rdrf/rdrf/dynamic_data.py

```diff
@@ -586,7 +586,8 @@ def handle_file_upload(registry_code, key, value, current_value):
             # Django uses a "clear" checkbox value of False to indicate file should be removed
             # we need to delete the file but not here
             logger.debug("User cleared %s - file will be deleted" % key)
-            to_delete = current_value
+            to_delete = True
```

Comment:
Can we make some minor improvements to the legacy code here:

1. Change to_delete as always Boolean. (Sometimes set as None)
2. Don't redefine value, use a separate rval var defined at the top of the function
3. Define to_delete as False at he top of the function, function scope, then we can remove the final else

These aren't comments on your code, rather on the legacy code, but it will only take a few mintues and we might as well tighten that up.

cheers.